src/main/java/com/google/cloud/tools/app/api/DefaultConfiguration.java

```diff
@@ -14,9 +14,7 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.app.impl.config;
-
-import com.google.cloud.tools.app.api.Configuration;
+package com.google.cloud.tools.app.api;
```

Comment:
i agree that the actual artifact name of this library needs work, but I
don't think that has to be tied to this PR.

On Mon, Jun 6, 2016 at 4:53 PM, meltsufin <notifications@github.com> wrote:

> In src/main/java/com/google/cloud/tools/app/api/DefaultConfiguration.java
> <https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/pull/105#discussion_r65966837>
> :
>
> > @@ -14,9 +14,7 @@
> >   * limitations under the License.
> >   */
> >
> > -package com.google.cloud.tools.app.impl.config;
> > -
> > -import com.google.cloud.tools.app.api.Configuration;
> > +package com.google.cloud.tools.app.api;
>
> I don't mind renaming app -> appengine, but in order to be consistent,
> wouldn't we also want to rename the library itself to
> appengine-tools-lib-for-java, not to mention the plugins?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/pull/105/files/5723a07a6bb84c29656eee4b39a1bb8a36e1b524#r65966837>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHf5HeigQs-IPiaAtMLK6fxGM7chXp14ks5qJIjLgaJpZM4IvDJS>
> .
>